modules/navier_stokes/include/base/NSFVBase.h

```diff
@@ -1931,15 +1942,29 @@ NSFVBase<BaseType>::addINSMomentumFrictionKernels()
 {
   unsigned int num_friction_blocks = _friction_blocks.size();
   unsigned int num_used_blocks = num_friction_blocks ? num_friction_blocks : 1;
+  const std::string u_names[3] = {"u", "v", "w"};
+  if (!parameters().isParamSetByUser("standard_friction_formulation"))
+  {
+    paramError("standard_friction_formulation", "User needs to define the friction formulation");
+  }
```

Comment:
Move this to `checkFrictionParameterErrors` and use the `checkDependentParameterError` function. Also this error check is not correct. As we discussed before you want to error when somebody declares these parameters with no friction available. It is still unprotected with this implementation. Also, considering that you have defaults for your parameters this error check is not needed.

Reply:
done